fe/src/main/java/org/apache/doris/load/TransactionCallBack.java

```diff
@@ -0,0 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.load;
+
+import org.apache.doris.transaction.TransactionState;
+
+public interface TransactionCallBack {
+
+    /**
+     * check txn has related job
+     * return true if txn has related job, return false if job of txn has been deleted
+     *
+     * @param txnState
+     * @return
+     */
+    boolean checkTxnHasRelatedJob(TransactionState txnState);
+
+    /**
+     * update catalog of job which has related txn after transaction has been committed
+     *
+     * @param txnState
+     */
+    void updateCatalogOnCommitted(TransactionState txnState);
```

Comment:
rename it to `onCommitted`, maybe some callback just want to print a notice other than update catalog